zap/src/test/java/org/zaproxy/zap/scan/filters/GenericFilterUtilityTest.java

```diff
@@ -0,0 +1,257 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.scan.filters;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.zaproxy.zap.WithConfigsTest;
+import org.zaproxy.zap.extension.ascan.filters.FilterCriteria;
+import org.zaproxy.zap.extension.ascan.filters.FilterResult;
+import org.zaproxy.zap.extension.ascan.filters.impl.AbstractGenericScanFilter;
+import org.zaproxy.zap.model.StructuralNode;
+
+/** @author KSASAN preetkaran20@gmail.com */
+public class GenericFilterUtilityTest extends WithConfigsTest {
+
+    private AbstractGenericScanFilter<String> abstractGenericScanFilter;
+
+    @Before
+    public void init() {
+        abstractGenericScanFilter =
+                new AbstractGenericScanFilter<String>() {
+
+                    @Override
+                    public FilterResult isFiltered(StructuralNode node) {
+                        // TODO Auto-generated method stub
+                        return null;
+                    }
+
+                    @Override
+                    public String getFilterType() {
+                        // TODO Auto-generated method stub
+                        return null;
+                    }
+                };
+    }
+
+    @Test
+    public void testEmptyNodeValuesAndFilterValuesIncludeCriteria() {
```

Comment:
The references to "node" should be removed.

Reply:
sure.